d1lod/d1lod/jobs.py

```diff
@@ -362,10 +365,10 @@ def export_graph():
     JOB_NAME = "EXPORT_GRAPH"
     logging.info("[%s] Job started.", JOB_NAME)
 
-    s = Store(SESAME_HOST, SESAME_PORT)
-    r = Repository(s, SESAME_REPOSITORY)
+    s = Store(VIRTUOSO_HOST, VIRTUOSO_PORT, VIRTUOSO_REPOSITORY, NAMESPACES)
+    r = Repository(host=VIRTUOSO_HOST, port=VIRTUOSO_PORT, name=VIRTUOSO_REPOSITORY, ns=NAMESPACES)
 
-    logging.info("[%s] Exporting graph of size %d.", JOB_NAME, r.size())
+    # logging.info("[%s] Exporting graph of size %d.", JOB_NAME, r.size())
```

Comment:
Could this either get implemented and put back in or just removed since we're using version control and don't really needed commented-out code.

Reply:
Resolved by implementing the `size()` function